index.html

```diff
@@ -27,21 +28,31 @@
   </head>
   <body>
     <main>
-        <header class="site-header clearfix">
+      <!-- LOGO SECTION -->
+  <section class="logo-section">
```

Comment:
Increase indentation. This is a child element of main, so should be tabbed in past it.